src/commentcnv.l

```diff
@@ -229,6 +231,32 @@ SLASHopt [/]*
                                       if (yyextra->lang!=SrcLangExt_Cpp) REJECT;
                                       copyToOutput(yyscanner,yytext,(int)yyleng);
                                     }
+<*>"<!--"                           {
```

Comment:
using `<*>` is risky, as it would also apply outside of comment blocks, i.e. when in the Scan state, which is undesired as someone could write

```c++
int k = "hey <!-- look at this";
```

Reply:
The `<*>` is indeed a bit dangerous.
I tested the given example together git this patch and it looks like it is working correctly.

I think it would probably be better not to use `<*>` but use a method analogous the one used with e.g. `\htmlonly` (i.e. consider it as a verbatim block).

I'll have a look at it....